### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by schweitzpgi**

### CODE REVIEW

File: flang/README.md [hunk 1]
```diff
@@ -46,100 +38,37 @@ This is quite similar to the old way, but with a few subtle differences.
...
 ```
 
 # How to Generate FIR Documentation
 
-If f18 was built with `-DLINK_WITH_FIR=On` (`On` by default), it is possible to
-generate FIR language documentation by running `make flang-doc`. This will
-create `docs/Dialect/FIRLangRef.md` in f18 build directory.
+If flang was built with `-DLINK_WITH_FIR=On` (`On` by default), it is
+possible to generate FIR language documentation by running `make
+flang-doc`. This will create `docs/Dialect/FIRLangRef.md` in f18 build
```
Comment (vjayathirtha-nv) [inline flang/README.md:73]: flang build directory?

File: flang/README.md [hunk 2]
```diff
@@ -8,34 +8,26 @@
 
 # FIR
 
-Working branch for FIR development.
+Yes, we've moved. This is now the working branch for FIR development.
```
Comment (vjayathirtha-nv) [inline flang/README.md:11]: is `fir-dev` going to be the default branch one sees in the `f18-llvm-project` repository? If not we don't need to have `Yes, we've moved` here, do we?

File: flang/README.md (hunk 1 above)
Comment (schweitzpgi) [reply flang/README.md:73]: Oh, we don't want the LINK_WITH_FIR garbage. Is that what you mean?

File: flang/README.md (hunk 2 above)
Comment (schweitzpgi) [reply flang/README.md:11]: It's not the default branch.

Comment (schweitzpgi) [reply flang/README.md:11]: I am planning to update https://github.com/schweitzpgi/f18/tree/mono to point here. The "we've moved" was intended to be a visual clue that people have gone to the correct place.

File: flang/README.md (hunk 1 above)
Comment (vjayathirtha-nv) [reply flang/README.md:73]: Line 71 changed `If f18 was built` -> `If flang was built`
Similarly line 73 should change `in f18 build directory` -> `in flang build directory`

File: flang/README.md (hunk 2 above)
Comment (vjayathirtha-nv) [reply flang/README.md:11]: I conflated this move with the f18 move to llvm monorepo. Disregard my comment.

File: flang/README.md (hunk 1 above)
Comment (schweitzpgi) [reply flang/README.md:73]: Looks like the documentation on `mono` got dorked.

Comment (schweitzpgi) [reply flang/README.md]: Fixed

File: flang/README.md [hunk 3]
```diff
@@ -46,100 +38,36 @@ This is quite similar to the old way, but with a few subtle differences.
...
-
-To run individual regression tests llvm-lit needs to know the lit
-configuration for f18. The parameters in charge of this are:
-flang_site_config and flang_config. And they can be set as shown bellow:
-```
-<path-to-llvm-lit>/llvm-lit \
- --param flang_site_config=<path-to-f18-build>/test-lit/lit.site.cfg.py \
- --param flang_config=<path-to-f18-build>/test-lit/lit.cfg.py \
-  <path-to-fortran-test>
+  cmake ../llvm/flang -DCMAKE_PREFIX_PATH=<llvm-instal-root> -DCMAKE_BUILD_TYPE=Release -DLLVM_TARGETS_TO_BUILD=X86 -DCMAKE_CXX_STANDARD=17 <other-arguments>
```
Comment (vjayathirtha-nv) [inline flang/README.md:64]: Is it worth mentioning that we can build it on non x86 targets too?

Comment (schweitzpgi) [reply flang/README.md:64]: This is just an introductory "getting started" sort of document. It's never gone to great lengths to explain any nuances. The assumption is that if you want to work on the tip of the FIR project, you gotta know what you're doing. :)

**Comment by schweitzpgi** [comment]
Can I get an approval?

**Pull request merged**